Source/ConversationV1/Conversation.swift

```
@@ -0,0 +1,94 @@
+            let userInfo = [NSLocalizedFailureReasonErrorKey: error]
+            return NSError(domain: domain, code: code, userInfo: userInfo)
+        } catch {
+            return nil
+        }
+    }
+    
+    /**
+     Start a new conversation or obtain a response for a submitted input message.
+     
+     - parameter message:     The user input message to send for processing.
+     - parameter context:     A dictionary that holds state, or context, for the conversation.
+     - parameter workspaceID: The conversation identifier. This is necessary.
+     - parameter failure:     A function executed if an error occurs.
+     - parameter success:     A function executed with the conversation application's response.
+     */
+    public func sendText(
+        message:     String! = nil,
+        context:     [String : JSON]?,
+        workspaceID: WorkspaceID,
+        failure:     (NSError -> Void)? = nil,
+        success:     MessageResponse -> Void)
+    {
+        
+        do {
+        let messageRequest = MessageRequest(message: message, context: context)
+        
+        // construct REST request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceURL + "/v1/workspaces/\(workspaceID)/message",
+            acceptType: "application/json",
+            contentType: "application/json",
+            messageBody: try messageRequest.toJSON().serialize()
+        )
+            
+            // execute REST request
+            Alamofire.request(request)
+                .authenticate(user: username, password: password)
+                .responseObject(dataToError: dataToError) {
+                    (response: Response<MessageResponse, NSError>) in
+                    switch response.result {
+                    case .Success(let response): success(response)
+                    case .Failure(let error): failure?(error)
+                    }
+            }
+        }
+        catch {
```

You can remove this catch statement if you switch to serializing the `messageRequest` as described in an earlier comment.

As an aside for future reference, though, we try to avoid print statements. We actually used to log a lot of messages to the console, but found that it cluttered the console and annoyed clients that were developing with the SDK. So now we prefer to return any information through the `failure` function instead of printing.